derex/runner/settings/derex/openedx_platform.py

```diff
@@ -0,0 +1,18 @@
+PLATFORM_NAME = "TestEdX"
+SITE_NAME = {"lms": "lms.localhost:4700", "cms": "studio.localhost:4700"}[
+    SERVICE_VARIANT
+]
+
+LMS_BASE = "http://lms.localhost:4700"
+CMS_BASE = "http://studio.localhost:4800"
```

Comment:
These values are variants of the ones in line 2.

They probably should come from the same place.